src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -248,6 +250,21 @@ private static IEnumerable<OngoingTask> CollectEtlTasks(DatabaseRecord databaseR
                         throw new InvalidOperationException(
                             $"Could not find connection string named '{ravenEtl.ConnectionStringName}' in the database record for '{ravenEtl.Name}' ETL");
 
+                    string url = null;
+                    string error = null;
+                    if (tag == store.NodeTag)
+                    {
+                        try
+                        {
+                            store.DatabasesLandlord.DatabasesCache.TryGetValue(databaseRecord.DatabaseName, out var task);
+                            url = ((RavenEtl)task.Result.EtlLoader.Processes.First(p=> p is RavenEtl etl && etl.Name == ravenEtl.Name)).Url;
```

Comment:
This is going to be really confusing to the user if there is any errors here.
You are handling it, but the error messages aren't going to be clear.